plugins/utils/surround.nix

```diff
@@ -5,10 +5,429 @@
   pkgs,
   ...
 }:
-helpers.vim-plugin.mkVimPlugin config {
-  name = "surround";
-  originalName = "surround.vim";
-  defaultPackage = pkgs.vimPlugins.surround;
```

Comment:
Is this functionally identical to the old surround.vim when options are left as defaults?